src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/store/ReadingDraftTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.draftstore.features.store;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.draftstore.SmokeTestSuite;
+import uk.gov.hmcts.reform.draftstore.response.Draft;
+import uk.gov.hmcts.reform.draftstore.response.DraftList;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestPropertySource("classpath:application.properties")
+@RunWith(SpringRunner.class)
+public class ReadingDraftTest extends SmokeTestSuite {
+
+    @Test
+    public void user_can_read_own_draft() throws IOException, JSONException {
+        String documentJson = "{\"testing\":\"functional\"}";
+        String draftId = draftStoreClient.createDraft(documentJson);
+        Optional<Draft> response = draftStoreClient.readDraft(draftId);
+        assertThat(response.isPresent()).isTrue();
+        Draft draft = response.get();
+        assertThat(draft.id).isEqualTo(draftId);
+        assertThat(draft.document).isNotNull();
```

Comment:
You could compare the whole documents, just like you do in one of the tests below.